urdf/CMakeLists.txt

```diff
@@ -27,7 +27,7 @@ catkin_package(
   LIBRARIES ${PROJECT_NAME}
   INCLUDE_DIRS include ${TinyXML_INLCLUDE_DIRS} ${CATKIN_DEVEL_PREFIX}/include
```

Comment:
Agreed.  I also believe that almost all of the includes listed in `include_directories()` are not needed, since they'll come in as part of `${catkin_include_DIRS}`.